lucene/core/src/java/org/apache/lucene/index/BufferedUpdates.java

```diff
@@ -80,6 +80,7 @@ load factor (say 2 * POINTER).  Entry is object w/
 
   private final Counter bytesUsed = Counter.newCounter(true);
   final Counter fieldUpdatesBytesUsed = Counter.newCounter(true);
+  private final Counter termsBytesUsed = Counter.newCounter();
```

Comment:
```suggestion
  private final Counter termsBytesUsed = Counter.newCounter(true);
```

Sorry I looked at it again and we have one case where we use it in a global context in the delete queue so we need the threadsafety. Rest looks great.